webpack.config.js

```diff
@@ -0,0 +1,88 @@
+const HtmlWebPackPlugin = require('html-webpack-plugin');
+const InterpolateHtmlPlugin = require('interpolate-html-plugin');
+const merge = require('webpack-merge');
+const path = require('path');
+const webpackExtension = mode => require(`./webpack.config.${mode}.js`);
+
+module.exports = ({ mode }) => {
+  return merge(
+    {
+      mode,
+      module: {
+        rules: [
+          {
+            test: /\.(js|jsx)$/,
+            exclude: /node_modules/,
+            use: {
+              loader: 'babel-loader',
+            },
+          },
+          {
+            test: /\.css$/i,
+            use: ['style-loader', 'css-loader'],
+          },
+          {
+            test: /\.(png|svg|jpg|gif|jpeg)$/,
+            use: [
+              {
+                loader: 'file-loader',
+                options: {
+                  esModule: false,
+                },
+              },
+            ],
+          },
+          {
+            test: /\.(woff|woff2|eot|ttf|otf)$/,
+            use: [
+              {
+                loader: 'file-loader',
+                options: {
+                  esModule: false,
+                },
+              },
+            ],
+          },
+          {
+            test: /\.(mov|mp4)$/,
+            use: [
+              {
+                loader: 'file-loader',
+                options: {
+                  name: '[name].[ext]',
+                },
+              },
+            ],
+          },
+          {
+            test: /\.html$/,
+            use: [
+              {
+                loader: 'html-loader',
+              },
+            ],
+          },
+        ],
+      },
+      plugins: [
+        new HtmlWebPackPlugin({
+          template: path.resolve(__dirname, 'public/index.html'),
+          filename: 'index.html',
+        }),
+        new InterpolateHtmlPlugin({
+          PUBLIC_URL: mode,
+        }),
+      ],
+      devServer: {
+        historyApiFallback: true,
+        port: 3000,
+      },
+      output: {
+        publicPath: '/',
```

Comment:
try setting 

publicPath : ""

Reply:
I guess the second option would work.

Reply:
any progress?

Reply:
Yes for image paths to work I had to import them and then use accordingly. Actually hard coded paths image paths in background-url cannot be taken care of by webpack, so I had to make them dynamic.

Reply:
Is it now working?

Reply:
Yes